lib/tasks/email_template.rake

```diff
@@ -0,0 +1,19 @@
+# rake db:email_template
+
+# Later, with more templates, email_template can be a namespace,
+# and each template could have separate task
+
+begin
```

Comment:
assuming there's a test for this, let's make sure that the output to STDOUT is muffled for the test run